jobwatcher/jobwatcher.py

```diff
@@ -176,27 +214,27 @@ def main():
 
         if pending > 0:
             # connect to asg
-            asg_conn = boto3.client('autoscaling', region_name=region, config=proxy_config)
+            asg_client = boto3.client('autoscaling', region_name=region, config=proxy_config)
 
             # get current limits
-            asg = asg_conn.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_name]).get('AutoScalingGroups')[0]
+            asg = asg_client.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_name]).get('AutoScalingGroups')[0]
 
-            min = asg.get('MinSize')
+            min_size = asg.get('MinSize')
             current_desired = asg.get('DesiredCapacity')
-            max = asg.get('MaxSize')
-            log.info("min/desired/max %d/%d/%d" % (min, current_desired, max))
+            max_size = asg.get('MaxSize')
+            log.info("min/desired/max %d/%d/%d" % (min_size, current_desired, max_size))
             log.info("Nodes requested %d, Nodes running %d" % (pending, running))
 
-            # check to make sure it's in limits
-            desired = running + pending
-            if desired > max:
-                log.info("%d requested nodes is greater than max %d. Requesting max %d." % (desired, max, max))
-                asg_conn.update_auto_scaling_group(AutoScalingGroupName=asg_name, DesiredCapacity=max)
-            elif desired <= current_desired:
-                log.info("%d nodes desired %d nodes in asg. Noop" % (desired, current_desired))
+            # Check to make sure requested number of instances is within ASG limits
+            required = running + pending
```

Comment:
I would prefer to keep "desired" (or new_desired) because we use this value to set the ASG desired capacity, and rather change the log string accordingly

Reply:
I used `required` in place of `desired` because it is the value that we need, but it will be used for the ASG only if it is lower than `max_size`.

Reply:
I changed the log messages to reflect the variable names.